src/frontend/citizen-portal/src/app/services/notice-of-dispute.service.ts

```diff
@@ -32,7 +32,7 @@ export class NoticeOfDisputeService {
     work_phone_number: [null, [FormControlValidators.phone]], // not using now
     email_address: [null, [Validators.required, Validators.email]],
     birthdate: [null, [Validators.required]],
-    drivers_licence_number: [null, [Validators.required]],
+    drivers_licence_number: [null, [Validators.required, Validators.minLength(7), Validators.maxLength(9)]],
```

Comment:
drivers_licence_province should be != "BC", it is better to be Validators.maxLength(20) or others
Maybe disable drivers_licence_number at first, enable it after drivers_licence_province is chosen

Reply:
as soon as dispute is retrieved, validators are reset